controllers/auth.js

```diff
@@ -18,8 +17,9 @@ module.exports = {
             }); 
         } else {
           // Check password
-          // const isMatch = await bcrypt.compare(results[0].password, password);
-          const isMatch = await results[0].password === password; 
+          const isMatch = await bcrypt.compare(password, results[0].password);
+
+          // const isMatch = await results[0].password === password; 
```

Comment:
Nice catch! I had done this to test the login because hashed passwords hadn't existed. Was going to call this out, but you found it before I got to it! Line 22 can likely be removed entirely as we won't be using plain text passwords.

Reply:
I figured as much :).  I've deleted line 22.